src/stories/foundation/colorDocs.tsx

```diff
@@ -0,0 +1,434 @@
+    <div className="gradients">
+      <header className="colorHeader">
+        <h3>Gradients</h3>
+        <p>
+          There are both regular and dark version of the gradients. The regular version is designed for black text on top, while the dark version is designed for white text on top.
+          <code>var(--ce-gradient-##)</code>
+        </p>
+      </header>
+      <h4 style={{ margin: '8px 0 4px' }}>Light (for dark text)</h4>
+      <div className="gradientCells">
+        {lightGradients.map((gradient) => (
+          <div className="colorCell" key={gradient}>
+            <div className="color" style={{ background: `var(--ce-gradient-${gradient})` }} />
+            <div className='grade'>
+              {gradient}
+            </div>
+          </div>
+        ))}
+      </div>
+      <h4 style={{ margin: '16px 0 4px' }}>Dark (for light text)</h4>
+      <div className="gradientCells">
+        {darkGradients.map((gradient) => (
+          <div className="colorCell" key={gradient}>
+            <div className="color" style={{ background: `var(--ce-gradient-${gradient})` }} />
+            <div className='grade'>
+              {gradient}
+            </div>
+          </div>
+        ))}
+      </div>
+    </div>
+  )
+}
+
+const ColorDocs: React.FC = () => {
+  return (
+    <div>
+      <h2>The "Magic Number" Color System</h2>
+      <p>In design systems compliant with WCAG 2.0 (and Section 508), the term <strong>"magic number"</strong> refers to the specific <strong>luminance difference</strong> required
+        between two colors to ensure they meet minimum contrast standards for accessibility.
+        The higher the magic number, the greater the contrast and accessibility.
+      </p>
+      <ul>
+        <li><strong>Magic number of 40+:</strong> Meets WCAG 2.0 AA Large Text (18pt/14pt bold or larger) contrast. For example, grey-90 and indigo-50.</li>
+        <li><strong>Magic number of 50+:</strong> Meets WCAG 2.0 AA contrast or AAA Large Text contrast. For example, grey-90 and red-40. <strong className="emphasis-text">We recommend this level of contrast for most web content.</strong></li>
+        <li><strong>Magic number of 70+:</strong> Meets WCAG 2.0 AAA contrast. For example, grey-10 and red-80.</li>
+      </ul>
+      <p>Additionally, a grade of 50 will meet the Section 508 AA contrast requirement against both pure white (#000) and pure black (#fff).</p>
+      <details>
+        <summary>More about A, AA, and AAA</summary>
+        <div className="details-content">
+          <p><strong>Always aim for AA.</strong> In some cases, A may be acceptable:</p>
+          <ul>
+            <li><strong>A:</strong> <em>Minimum</em> level. Text must have a contrast ratio of at least <b>3:1</b> for large text (18pt/14pt bold or larger) and <b>4.5:1</b> for normal text against its background. This level is the basic requirement for accessibility, ensuring that text is readable for most users with mild visual impairments.</li>
```

Comment:
This is well-written and explains it thoroughly, but I wonder if we really need/want to go into this level of detail here. 

If they already understand accessibility guidelines, it's redundant and vulnerable to falling out of date.

If they don't, a simple "Always aim for AA" is probably sufficient.

Reply:
I'd like to leave it then, because it helps contextualize the magic numbers without having to go google the A-AAA